tests/e2e/specs/wcpay/merchant/merchant-admin-analytics.spec.js

```diff
@@ -36,10 +36,9 @@ describe( 'Admin Order Analytics', () => {
 		// Go to Action Scheduler
 		await merchantWCP.openActionScheduler( 'pending', orderId );
 		const importOrderRun = await page.$x(
-			'//code[contains(text(), "0 => ' +
-				orderId +
-				'")]/ancestor::tr//span[contains(@class, "run")]/a'
+			`//tr[contains(., "0 => ${ orderId }") and contains(., "${ actionSchedulerHook }")]//span[contains(@class, "run")]/a`
```

Comment:
In the case of an Atomic site (having different plugins enabled by default), it is necessary to be more specific in the selector with the action's name, because there usually is another action at the top of the list (coming from AutomateWoo for instance).